modules/ensemble/lib/widget/input/slider.dart

```diff
@@ -36,8 +35,8 @@ class EnsembleSlider extends StatefulWidget
   @override
   Map<String, Function> setters() {
     return {
-      'initialValue': (value) =>
-          _controller.value = Utils.optionalDouble(value) ?? 0,
+      'initialValue': (value) => _controller.value =
+          Utils.getRangeValues(value) ?? const RangeValues(0, 0),
```

Comment:
The getter implementation for `RangeValues` needs to expose the individual `start` and `end` values since returning only the RangeValues object doesn't allow direct access to these values.

Reply:
I intentionally did not do that. Since we are using RangeValue for for both, a user can get start or end value through `value`. 
A thing we can do instead is make value as single `value` getter (in case start == end), and also add `start` and `end` getters. wdys?